model/vcs/registry.yaml

```
@@ -239,3 +239,26 @@ groups:
         brief: >
           The state of the change (pull request/merge request/changelist).
         examples: ["open", "closed", "merged"]
+      - id: vcs.owner.name
+        type: string
+        stability: development
+        brief: >
+           The group owner within the version control system.
+        examples:
+          [
+            "my-org",
+            "myteam",
+            "business-unit",
+          ]
+      - id: vcs.vendor.name
```

provider would be a broader and more neutral term here. We also have some examples of `*.provider.name` (e.g. feature_flag.provider.name, https://github.com/open-telemetry/semantic-conventions/pull/1982) 

```suggestion
      - id: vcs.provider.name
```

---

`provider` sounds good, especially if there's consistency there w/ other attributes.

---

Thanks for review, updated